### ISSUE TITLE
How to extract “validation dataset” from "final_model_dataset/train.fa"

### ISSUE THREAD

**Issue by NickShanyt** [f-kretschmer/bertax_training#11]
Hi,
I have already downloaded the `"final_model_dataset"` and got `"test.fa"` & `"train.fa"`, and I know from the paper that `"test.fa"` will extract 2000 seqs per phylum, also it mentioned that "From the remaining samples, we select 95% of the samples of each phylum as the training set and use the remaining 5% as the validation set (Fig. 4A)".

I want to know how I could extract "95% of the samples of each phylum as the `training set` and use the remaining 5% as the `validation set`", I noticed that some scripts in this repository may help, but I don't know how to use them.

I would be grateful if you could tell me what to do.

**Comment by flomock** [comment]
As the validation set is only used to decide whether or not to stop training, we randomly selected 5% of the training set as the validation set at the start of training. So for each training run we started, the validation set was different. So if you wanted to do the same thing, you could do something like this:
```python
from sklearn.model_selection import train_test_split
training_set, validation_set = train_test_split(training_set, test_size=0.05, random_state=42)
```
